capstan.go

```diff
@@ -423,6 +423,7 @@ func main() {
 						cli.BoolFlag{Name: "pull-missing, p", Usage: "attempt to pull packages missing from a local repository"},
 						cli.StringFlag{Name: "boot", Usage: "specify default config_set name to boot unikernel with"},
 						cli.StringSliceFlag{Name: "env", Value: new(cli.StringSlice), Usage: "specify value of environment variable e.g. PORT=8000 (repeatable)"},
+						cli.StringFlag{Name:"remote-guest-name",Usage:"specify hostname of remote guest"},
```

Comment:
Perhaps using short name `remote` would do better? I think users will use IP more often than the hostname. Something like this:

```bash
$ capstan package compose --remote 189.16.0.5
```